src/it/java/org/eclipse/microprofile/starter/utils/Logs.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2017-2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertFalse;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public class Logs {
+    private static final Logger LOGGER = Logger.getLogger(Logs.class.getName());
+
+    public static final String S = File.separator;
```

Comment:
Reassigning a constant to another constant is not very useful.

Reply:
I used it as a shorthand with 
```..."See target" + S + "archived-logs" + S + testClass + S + testMethod ...```
Just a convenience... 

What would you suggest?

Reply:
Use `File.separator` directly in the code instead of `S`. As people don't know what S is but do now File.separator.